R/oc_process.R

```diff
@@ -0,0 +1,151 @@
+#' Process OpenCage geocoding request
+#'
+#' This function processes all geocoding requests issued by
+#' \code{\link{oc_forward}} and \code{\link{oc_reverse}} by calling the
+#' respective functions (after the query arguments have been checked by
+#' \code{\link{oc_check_query}}). It builds the URL, fetches the results,
+#' checks the status of the returned results and finally parses them.
+#'
+#' @param limit The maximum number of results that should be returned. Integer
+#'   values between 1 and 100 are allowed, the default is 1.
+#' @inheritParams oc_forward
+#'
+#' @return \code{oc_forward} returns, depending on the \code{return} parameter,
+#'   a list with either
+#'   \itemize{
+#'   \item the results as tibbles (\code{"df_list"}, the default),
+#'   \item the results as JSON lists (\code{"json_list"}),
+#'   \item the results as GeoJSON lists (\code{"geojson_list"}), or
+#'   \item the URL of the OpenCage API call for debugging purposes
+#'   (\code{"url_only"}).
+#'   }
+#'
+#' @keywords internal
```

Comment:
same as for other internal functions, I think it'd be better to have it as `@noRd`

Reply:
Again `@inheritParams oc_forward`. My preference isn't strong though.

Reply:
I think the parameter inheritance would still work? If not a good question to ask in RStudio community 🤔